e2e/README.md

```diff
@@ -412,35 +388,12 @@ For now, our best practices exist as self-tested, living documentation in our [e
 
 For best practices with regards to mocking network responses, see our [couchdb.e2e.spec.js](./tests/functional/couchdb.e2e.spec.js) file.
 
-### Tips & Tricks
+### Tips & Tricks (TODO)
```

Comment:
I think we're accidentally reverting changes here? Might need to rebase your branch onto master